webapp/src/ts/actions/contacts.ts

```diff
@@ -9,6 +9,7 @@ export const Actions = {
   selectContact: createMultiValueAction('SELECT_CONTACT'),
   setSelectedContact: createSingleValueAction('SET_SELECTED_CONTACT', 'selected'),
   setContactsLoadingSummary: createSingleValueAction('SET_CONTACT_LOADING_SUMMARY', 'value'),
+  setContactIdToFetch: createSingleValueAction('SET_CONTACT_ID_TO_FETCH', 'id'),
```

Comment:
For consistency (we don't use the term "fetch" anywhere else), could you please rename this to something like `setContactIdToSelect` ?

Reply:
Yeah, the name is weird. I want to make a difference when it looks for contact data and when it sets the contact as "selected" once we have the data so it's more straightforward for devs when it's used. 

Maybe `setContactIdToFind`? or `setContactIdToSummon`? 😅

Reply:
`setContactIdToLoad` ?

Reply:
Done